contracts/Controller.sol

```diff
@@ -161,43 +161,13 @@ contract Controller is ReentrancyGuard, Ownable {
         address oracleModule = AddressBookInterface(addressBook).getOracle();
         OracleInterface oracle = OracleInterface(oracleModule);
 
-        (bytes32 batch, , , , uint256 otokenExpiryTimestamp) = getBatchDetails(_otoken);
-        return oracle.isDisputePeriodOver(batch, otokenExpiryTimestamp);
-    }
-
-    /**
-     * @notice get batch for a specific otoken address
-     * @dev batch is the hash of option underlying, strike, collateral assets and the expiry timestamp
-     * @param _otoken otoken address
-     * @return batch hash in bytes32, batch underlying asset, batch strike asset, batch collateral asset, batch expiry timestamp
-     */
-    function getBatchDetails(address _otoken)
```

Comment:
Why removed this function ?

Reply:
we don't need batch ID in the whole system anymore, so I don't see how would we need this function for